docs/src/main/paradox/openshift.md

```diff
@@ -51,24 +59,17 @@ Container Registry in Lightbend's OpenShift cluster, CentralPark. You will have
 ```bash
 sbt docker:publishLocal
 
-export DOCKER_REGISTRY_SERVER=docker-registry-default.centralpark.lightbend.com
-export DOCKER_REGISTRY=$DOCKER_REGISTRY_SERVER/$OPENSHIFT_PROJECT
-
 docker login -p $TOKEN -u unused $DOCKER_REGISTRY_SERVER
 docker tag $IMAGE:$TAG $DOCKER_REGISTRY/$IMAGE:$TAG
 docker push $DOCKER_REGISTRY/$IMAGE:$TAG
-```
 
-Apply your deployment : 
-
-```bash
-$ oc apply -f deployment/openshift-play-scala-grpc-example.yml
-deployment.apps "play-scala-grpc-example-v1-0-snapshot" configured
-service "play-scala-grpc-example" unchanged
-route.route.openshift.io "play-scala-grpc-route" created
+## The `kustomize` step uses a `kustomization.yml` prepared for $DOCKER_REGISTRY/$IMAGE:$TAG. You will have to
+## create your own `deployment/overlays` folder (make a copy of `deployment/overlays/centralpark` and edit
+## `kustomization.yml`)
+kustomize build deployment/overlays/centralpark | oc apply -f -
```

Comment:
```suggestion
kustomize build deployment/overlays/openshift | oc apply -f -
```

or 

```suggestion
oc apply -f deployment/openshift
```

Reply:
No, this overlay is 100% bound to centralpark. It's not a generic openshift YML. That's the point. :-)